vm-setup/roles/v1aX_integration_test/templates/cluster-template-controlplane.yaml

```diff
@@ -78,7 +78,9 @@ spec:
         - type: phy
           id: enp1s0
           macAddress:
+{% if IMAGE_OS != "Centos" %}
             fromHostInterface: enp1s0
+{% endif %}
```

Comment:
There should not be a condition here. you need the fromHostInterface in both cases

Reply:
I am sorry, but it is a bit confusing now, is not `enp1s0` specific to Ubuntu as Kashif was mentioning earlier in the comments. So I was conditioning it to make it work only for specific OS.

Reply:
what Mael is suggesting is `enp1s0` is not specific to Ubuntu but its coming from IPA, so ignore my comments

Reply:
Thanks for clarification sir!